src/main/java/org/folio/circulationbff/controller/CirculationBffController.java

```diff
@@ -132,6 +136,17 @@ public ResponseEntity<MediatedRequest> saveAndConfirmMediatedRequest(
     }
   }
 
+  @Override
+  public ResponseEntity<CirculationLoans> findCirculationLoansByQuery(
+    String query, Integer limit, Integer offset, String totalRecords) {
```

Comment:
```suggestion
  public ResponseEntity<CirculationLoans> findCirculationLoansByQuery(String query, Integer limit, 
    Integer offset, String totalRecords) {
```

Reply:
Won't apply it, it's my readable with initial formatting

Reply:
you can simply add a line break when the line reaches the character limit, like this:
```
return ResponseEntity.ok(circulationLoanService.findCirculationLoans(query, limit, offset, 
  totalRecords));
```

Reply:
I fixed this issue, let's resolve it, it's the same number of lines and only a personal (subjective) view on how it must look like. It's not related to a functionality

Reply:
Also, if you have some best practices for improving code, you can document them somewhere because some internal team members' rules are not known outside of Vega team developers.
For example: https://folio-org.atlassian.net/wiki/spaces/FOLIJET/pages/509149235/Eureka+Code+Uniformity